app/components/game-card.js

```diff
@@ -30,6 +36,30 @@ export default Ember.Component.extend({
   /** @property {Boolean} only interact with the cards if you're player one */
   readOnly: false,
 
+  /** Attach document key handlers for magnifying purposes. */
+  attachKeyHandlers: Ember.on('init', function () {
```

Comment:
Not too sure. An alternative approach could set one set of key listeners on the game controller and bind a property to all game cards.